library/std/src/env.rs

```diff
@@ -236,22 +236,14 @@ fn _var(key: &OsStr) -> Result<String, VarError> {
 }
 
 /// Fetches the environment variable `key` from the current process, returning
-/// [`None`] if the variable isn't set or there's another error.
+/// [`None`] if the environment variable isn't set. It may return `None` also
```

Comment:
Note, the first paragraph is best as a relatively short "title", as that's what will be listed on the module page. Can you break this up?